WorkloadManagementSystem/Client/Matcher.py

```diff
@@ -311,8 +312,8 @@ def _checkCredentials(self, resourceDict, credDict):
       # You can only match groups in the same VO
       if credDict['group'] == "hosts":
         # for the host case the VirtualOrganization parameter
-        # is mandatory in resourceDict
```

Comment:
I am not sure it is correct. We use host certificate for cloud VM's pilots. In this case the VirtualOrganization option is passed as part of the Resource description because the host certificate is neutral with this resect

Reply:
See https://github.com/DIRACGrid/VMDIRAC/blob/9c6ef192d400686431e42c09de7086d49dda7575/VMDIRAC/WorkloadManagementSystem/Bootstrap/vm-pilot#L124

Reply:
OK thanks. I revert these changes.